pkg/antctl/raw/supportbundle/command_test.go

```diff
@@ -351,17 +408,36 @@ func TestProcessResults(t *testing.T) {
 				defaultFS = afero.NewOsFs()
 			}()
 
-			err := processResults(tt.resultMap, option.dir)
-			if tt.expectedErr != "" {
-				require.ErrorContains(t, err, tt.expectedErr)
-			} else {
-				require.NoError(t, err)
-			}
-			// Both test cases above have failed Nodes, hence this file should always be created/
+			antreaInterface := fakeclientset.NewSimpleClientset(&controllerInfo, agentInfo1, agentInfo2)
+			k8sClient := fake.NewSimpleClientset(controllerPod, pod1, pod2)
+			err := processResults(context.TODO(), antreaInterface, k8sClient, tt.resultMap, option.dir)
+			require.NoError(t, err)
 			b, err := afero.ReadFile(defaultFS, filepath.Join(option.dir, "failed_nodes"))
 			require.NoError(t, err)
 			data := string(b)
 			for node, err := range tt.resultMap {
+				tgzFileName := fmt.Sprintf("agent_%s.tar.gz", node)
+				if node == "" {
+					tgzFileName = "controller_node-1.tar.gz"
+				}
+				if err != nil {
```

Comment:
Add a comment to indicate that this is using the "fallback" path, since collecting the bundle through the Antrea API failed